app/services/fetch_projects.rb

```diff
@@ -72,8 +72,8 @@ def call
     end
   end
 
-  private def get_user_assignment(user)
     assignments = tenk.users.assignments.list(
+  private def current_user_assignments(user)
```

Comment:
> It looks like this method name change has added it onto a different line - 76 rather than 75. I think it a positive trait to have commits that are self-contained, rather than being dependent on the next commit to fix the history.

🙈 🙈 🙈 I did ALL of the changes, and then tried to pick them apart to commit separately. It... seems to have backfired. 🙈